### DISCUSSION TITLE
weblab environment creation

### DISCUSSION THREAD

**Discussion by mwandu-byte** [The-Art-of-Hacking/h4cker#239] [General]
how do I create my own penetration test environment

**Comment by mwandu-byte** [comment]
I need help please

**Reply by santosomar** [reply]
The following are some tips and instructions on how you can build your own lab for penetration testing and to practice different defensive techniques helpful for incident response and digital forensics.

https://github.com/The-Art-of-Hacking/h4cker/tree/master/build_your_own_lab
